CHANGELOG-core.md

```diff
@@ -25,6 +26,11 @@ This release contains a number of features aimed at improving the node's reliabi
 - `ETH_FINALITY_DEPTH` specifies how deep protection should be against re-orgs. The default is 50. It only applies if BulletproofTxManager is enabled. It is not recommended to change this setting.
 - `EthHeadTrackerHistoryDepth` specifies how many heads the head tracker should keep in the database. The default is 100. It is not recommended to change this setting.
 
+### Breaking changes
+
+- `admin withdraw` command has been removed. This was only ever useful to withdraw LINK if the Oracle contract was owned by the Chainlink node address. It is no longer recommended to have the Oracle owner be the chainlink node address.
```

Comment:
```suggestion
- `admin withdraw` command has been removed. This was only ever useful to withdraw LINK if the Oracle contract was owned by the Chainlink node address. It is recommended to have the Oracle owner be a different address than the Chainlink node address.
```
It was never recommended to have the Oracle owner be the Chainlink node address, it was only available as an option.

Reply:
Thanks @thodges-gh